java/src/main/java/io/rapidpro/expressions/utils/ExpressionUtils.java

```diff
@@ -2,6 +2,8 @@
 
 import io.rapidpro.expressions.dates.DateStyle;
 import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.CharEncoding;
```

Comment:
This is what happens when I don't have flake8 to help and I forget that my IDE automatically adds imports for me!